### PR TITLE
Add WooCommerce options

### PR DESCRIPTION
**Pull request by akshitsethi**
This PR adds WooCommerce options to the plugin.

### CODE REVIEW

File: src/woocart-defaults.php [hunk 1]
```diff
@@ -137,7 +170,7 @@ public function load_textdomain() {
      */
     public static function activate_plugin() {
         // Add to `wp_options` table.
-        update_option( self::OPTIONNAME, WP_CONTENT_DIR . '/config' );
+        update_option( 'WooCartDefaults.Path', WP_CONTENT_DIR . '/config' );
```
Comment (dz0ny) [inline src/woocart-defaults.php]: Needs to go out, bundles are never public...

File: src/woocart-defaults.php [hunk 2]
```diff
@@ -41,7 +41,7 @@
  */
 class WooCartDefaults {
 
-    const OPTIONNAME = 'WooCartDefaults.Path';
+    const SETTINGNAME   = 'WooCartDefaults.Settings';
```
Comment (dz0ny) [inline src/woocart-defaults.php]: Do we even use this?

Comment (akshitsethi) [reply src/woocart-defaults.php]: We use the `settings` option where we store the `keys` for the settings in the database.

Comment (dz0ny) [reply src/woocart-defaults.php]: But do we need to store those, we fetching them from defaults file?

Comment (akshitsethi) [reply src/woocart-defaults.php]: We don’t if we are never going to export the options. Will remove them in the next fix I am going to push.

**Pull request merged**